recipe/patches/0001-backend_torch.patch

```diff
@@ -0,0 +1,15 @@
+diff --git a/keras/backend/config.py b/keras/backend/config.py
+index 19af01fe8..15618cfe0 100644
+--- a/keras/backend/config.py
++++ b/keras/backend/config.py
+@@ -12,8 +12,8 @@ _EPSILON = 1e-7
+ # Default image data format, one of "channels_last", "channels_first".
+ _IMAGE_DATA_FORMAT = "channels_last"
+ 
+-# Default backend: TensorFlow.
+-_BACKEND = "tensorflow"
++# Default backend: pytorch.
++_BACKEND = "torch"
```

Comment:
Can I ask the reasoning behind using this patch instead of an activation script that sets `KERAS_BACKEND`?

Reply:
I don't believe there was a specific reason. Just came to mind first. Is there a difference between using a patch or using an activation script in this case?

Reply:
Thinking through it a bit more, a patch might be preferable, since an activation script could overwrite the user's settings. I'm generally more inclined to leave it alone and let the user set it appropriately, but considering the request to change the default, this seems the least intrusive way to do so.

Reply:
I think activation scripts are discouraged if there's another solution: https://docs.conda.io/projects/conda-build/en/latest/resources/activate-scripts.html#activate-scripts

Reply:
I agree with @danpetry about the activation scripts. 

Previously, there were present but they were not utilized (not copied, not loaded), therefore we removed them. See https://github.com/AnacondaRecipes/keras-feedstock/pull/3/commits/d04fa2c920019ea0a06e62587f96aca41e6dab5e .

With this solution, we only set as default backend `torch`. The user can still override this choice using the environment variable, or the existing config.

Note: SF requested `keras` with `torch`.

Reply:
resolved, default backend kept as upstream (tensorflow).